clients/csharp-wpf/README.md

```diff
@@ -47,17 +47,18 @@ Let's review the hardware, software, and subscriptions that you'll need to use t
    ```
     -  Launch Visual Studio 2017 or newer by opening the solution `VoiceAssistantClient.sln`. Build the solution (the default build flavor is Debug x64)
     - Run the executable. For example, for Debug x64 build, this will be the executable: `VoiceAssistantClient\bin\x64\Debug\VoiceAssistantClient.exe`.
-1. When you first run the application, the **Setting** page will open. The first two fields are required (all others are optional):
-    - Enter _Subscription key_. This is your Azure [Speech Services Key](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/get-started)
-    - Enter _Subscription key region_. This is the Azure region of your key in the format specified by the "Speech SDK Parameter" column [in this table](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/regions#speech-to-text-text-to-speech-and-translation) (for example "westus")
-    - Leave the field _Custom commands app Id_ empty (unless you plan to use [Custom Commands](#use-custom-commands))
+1. When you first run the application, the **Setting** page will open. The first three fields are required (all others are optional):
+    - Enter _Connection Profile_. A name of your choice to identify this connection. The tool remembers multiple connection profiles so you can easily switch between them.
+    - Enter _Subscription key_. This is your Azure [Speech Services Key](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/get-started).
+    - Enter _Subscription key region_. This is the Azure region of your key in the format specified by the "Speech SDK Parameter" column [in this table](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/regions#speech-to-text-text-to-speech-and-translation) (for example "westus").
+    - Leave the field _Custom commands app Id_ empty (unless you plan to use [Custom Commands](#use-custom-commands)).
     - The default input language is "en-us" (US English). Update the _Language_ field as needed to select a different [language code from the "Speech-to-text" list](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/language-support).
-    - Press _Ok_ when you're done.
-    - Your entries will be saved and populated automatically when you launch the app again.
+    - Press _Save and Apply Profile_ when you're done.
+    - Your entries will be saved under this profile name and will be available when you launch the app again.
 ![Setting page](docs/media/SettingsPage.png)  
 1. Press **Reconnect**. The application will try to connect to your bot via Direct Line Speech channel.
 ![Main Page](docs/media/MainPage.png)

```

Comment:
This image is incorrect - it is a screen capture before Reconnect has been clicked. 

After pressing reconnect and a successful connection, The Connection Profile: **profilename** is displayed and message text box says "New conversation started - type or press the microphone button" --- also depending on the Bot, Activity Log may have an activity to display

Reply:
Also, the Snip & Sketch tool has a ruler option so your lines can look symmetrical ☺️

Reply:
I replaced the image was one that is captured AFTER the reconnection, and mentioned that the profile name appears at the top. 
I'm still using the only snipping tool! switching to the new scratch & snip I see the ruler. So cool! updated the images.